salvo-remote/sandboxes/README.md

```diff
@@ -0,0 +1,73 @@
+# Sandboxes for remote Salvo executions
+
+This directory contains definition of Salvo sandboxes. A sandbox is the
+collection of resources needed for a single Salvo execution. These resources
+are deployed in AWS. At the minimum a sandbox will contain:
+
+- a VM running Nighthawk (the load generator).
+- a VM running Envoy (the system under test or SUT).
+- a VM running the Nighthawk test server (the test backend).
+
+Note that the subnets used between the components are part of the common
+infrastructure and are already deployed as part of the [VPC
+configuration](https://github.com/envoyproxy/ci-infra/blob/main/salvo-infra/vpc.tf)
+done in the
+[ci-infra](https://github.com/envoyproxy/ci-infra/tree/main/salvo-infra)
+repository.
+
+Various sandbox types differ in the amount of instances deployed, amount of
+support systems deployed and the topology.
+
+# Available Sandbox types
+
+## The default sandbox
+
+The default contains the bare minimum of components needed to execute Salvo
+tests. The diagram below outlines the topology and content of the default
+sandbox. The default sandbox is currently available for the `x64` architecture
+only.
+
+![Diagram 1 - the default sandbox](images/the_default_sandbox.png)
+
+# Sandbox creation
+
+When Salvo executes, the CI pipeline will produce binaries and VM disk images
+for all the components. Once that is done, a new CI job is executed in the
+`salvo-control` agent pool on AZP. This job is then picked up by a Salvo
+control VM running a Salvo controller that instruments the sandbox creation.
+
+The Salvo controller uses the Terraform templates found in this directory,
+to deploy the sandbox components in AWS.
+
+# Sandbox instance life-cycle
+
+The Terraform configuration for sandboxes uses shared state bucket deployed in
+S3 on AWS. The sandbox instances that will be deployed or destroyed are
+determined based on the variables passed to Terraform.
+
+Each of the following variables is a list of build IDs. Each list represents
+one sandbox type. Each listed build ID represents an instance of the sandbox
+type that should be deployed. The build IDs must match the AZP build ID of the
```

Comment:
I'm a little bit confused about it, what is "AZP build ID", is it the `salvo-control` agent ID?

Reply:
Thank you for raising this concern, the text wasn't very clear. I have reworded it and added a clear description of what a `build ID` is. Please reopen if some concepts still need more explanation.

Done in bd193a8a279359b6fd7f1e16756b996730998576.